django/db/migrations/operations/models.py

```diff
@@ -821,6 +851,9 @@ def deconstruct(self):
     def describe(self):
         return 'Create constraint %s on model %s' % (self.constraint.name, self.model_name)
 
+    def suggest_migration_name(self):
+        return "%s_%s" % (self.model_name_lower, self.constraint.name.lower())
```

Comment:
`AddConstraint()`/`RemoveConstraint()` patterns should be consistent with `AddIndex()/RemoveIndex()`. We can mention a model name in both cases.

Reply:
I agree